firebase-inappmessaging/src/test/java/com/google/firebase/inappmessaging/internal/InAppMessageStreamManagerTest.java

```diff
@@ -107,10 +107,10 @@
       new Schedulers(trampoline(), trampoline(), trampoline());
 
   private static final CampaignImpressionList CAMPAIGN_IMPRESSIONS =
-      CampaignImpressionList.newBuilder()
-          .addAlreadySeenCampaigns(
-              CampaignImpression.newBuilder().setCampaignId(CAMPAIGN_ID_STRING).build())
-          .build();
+          CampaignImpressionList.newBuilder()
+              .addAlreadySeenCampaigns(
+                      CampaignImpression.newBuilder().setCampaignId(CAMPAIGN_ID_STRING).build())
+              .build();
```

Comment:
I wish these lines could not be marked as changed :( everywhere git blame is now going to say "Rohan did it"